presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -154,6 +154,19 @@ void getData(
 
         promiseHolder->promise.setValue(std::move(result));
       });
+
+  if (!bufferFound) {
+    // Buffer was erased for current TaskId.
+    VLOG(1) << "Task " << taskId << ", buffer " << bufferId << ", sequence "
+            << token << " Results size: "
```

Comment:
There isn't a value after "Results size: ". Do you want to add "0" for number of bytes or print something more meaningful ? You could remove the phrase also.

Reply:
Do you want to update this message ?